app/models/application_record.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class ApplicationRecord < ActiveRecord::Base
-  primary_abstract_class
```

Comment:
According to [official docs](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Inheritance/ClassMethods.html#method-i-primary_abstract_class)(click on "Source: show") the `primary_abstract_class` method does invoke `self.abstract_class = true`. I'm not sure about other part of this method tho, Rails 7 still sometimes surprises me 😅 

Are you sure you need this change?

Reply:
Yes, primary_abstract_class works too